README.md

```diff
@@ -26,3 +26,50 @@
 Laravel\Fortify\FortifyServiceProvider::class,
 ARKEcosystem\Fortify\FortifyServiceProvider::class,
 ```
+
+5. Add a `last_login_at` timestamp to your users table.
```

Comment:
Doesn't seem to be necessary to note this because https://github.com/ArkEcosystem/laravel-fortify/pull/6/files#diff-51885f76a0d5c572c6f400dfaaf1ec52fcdbeb03d2a4533e15c0175ef20bb680R17 is present?